nextcord/interactions.py

```diff
@@ -359,6 +364,43 @@ async def delete_original_message(self) -> None:
             session=self._session,
         )
 
+    async def send(self, *args, **kwargs) -> Optional[Message]:
+        """|coro|
+
+        This is a fallback function for help when sending messages in
+        response to an interaction. If the response
+        :property:`InteractionResponse.is_done()` then a messahe os sent
+        to the :property:`Interaction.channel` instead.
+
+        .. warning::
+
+            Ephemeral messages should not be sent with this as if
+            the :property:`Interaction.channel` is fallen back to,
+            ephemeral messages cannot be sent with this.
+
+        Returns
+        -------
+        Optional[:class:`Message`]
+            Message if the interaction has been responded to and the
+            interaction's channel was sent to. Else ``None``
+
+        Raises
+        ------
+        InvalidData
+            Somehow :property:`Interaction.channel` was ``None``,
+            this may occur in threads.
```

Comment:
Correct me if I'm wrong but I haven't seen `` :property:`...` `` be used, is it `` :attr:`...` ``?

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html?highlight=py%3Aproperty#directive-py-property

Reply:
sphinx docs seem to be incorrect as ``docstring of nextcord.interactions.Interaction.send:18: WARNING: Unknown interpreted text role "property".``